reference/platform-app-yaml.md

```diff
@@ -174,7 +174,7 @@ It has a few subkeys which are:
 
 #### Locations
 
-The `locations` key allows you to provide specific parameters for different URL prefixes.
+The `locations` key allows you to provide specific parameters for different URL prefixes. Each entry's key is an absolute URL path, and its value is configuration directives for that path.  That is, if your domain is `example.com` then `"/"` means "requests for `example.com/`", while `"/admin"` means "requests for `example.com/admin`".
```

Comment:
According to https://tools.ietf.org/html/rfc3986#section-4.2

> A relative reference that begins with a single slash character is
   termed an absolute-path reference.  A relative reference that does
   not begin with a slash character is termed a relative-path reference.

Shall we replace "an absolute URL path" with either "an absolute-path reference" or "a relative URL using absolute-path reference"?